actions/reconfigure.go

```diff
@@ -153,7 +153,7 @@ func (m *Reconfigure) reloadFromRegistry(addresses []string, instanceName, mode
 		var data map[string]interface{}
 		json.Unmarshal(body, &data)
 		count = len(data)
-		for key, _ := range data {
+		for key := range data {
```

Comment:
This was actually done by gofmt. Was using sublime and was formatting all the files I was changing. So instead of relying on that - I ran gofmt on the tree to make sure the formatting was clean.